package.json

```
@@ -21,26 +21,26 @@
     "email": "dev@cordova.apache.org"
   },
   "dependencies": {
-    "cordova-common": "^2.2.0",
-    "fs-extra": "^6.0.1",
+    "cordova-common": "^3.0.0",
+    "fs-extra": "^7.0.0",
     "npm-package-arg": "^6.1.0",
     "pify": "^4.0.0",
     "resolve": "^1.8.1",
     "semver": "^5.5.0",
     "which": "^1.3.1"
   },
   "devDependencies": {
-    "eslint": "^4.19.1",
-    "eslint-config-semistandard": "^11.0.0",
-    "eslint-config-standard": "^10.2.1",
+    "eslint": "^5.8.0",
+    "eslint-config-semistandard": "^13.0.0",
+    "eslint-config-standard": "^12.0.0",
     "eslint-plugin-import": "^2.13.0",
-    "eslint-plugin-node": "^5.2.1",
-    "eslint-plugin-promise": "^3.8.0",
-    "eslint-plugin-standard": "^3.1.0",
+    "eslint-plugin-node": "^8.0.0",
+    "eslint-plugin-promise": "^4.0.0",
+    "eslint-plugin-standard": "^4.0.0",
     "file-url": "^2.0.2",
-    "jasmine": "^2.4.1",
+    "jasmine": "^3.1.0",
```

```suggestion
    "jasmine": "^3.3.0",
```

(I would favor making it explicit that we would now expect at least version 3.3.0)